configs/teres_i_defconfig

```diff
@@ -0,0 +1,35 @@
+CONFIG_ARM=y
```

Comment:
Do 'make savedefconfig' and use produced 'defconfig'. Yours has a lot of garbage, e.g. there's no AXP803 driver in u-boot, so CONFIG_AXP_DLDO2_VOLT has no effect. See pinebook defconfig for a reference.